app/enterprise/1.5.x/kong-for-kubernetes/install-on-kubernetes.md

```diff
@@ -0,0 +1,306 @@
+    |`env.password.valueFrom.secretKeyRef.key` | The type of secret key used for authentication. If you followed the default settings in the example above, this is `password`. |
+    |`image.repository` | The Docker repository. In this case, `kong-docker-kong-enterprise-edition-docker.bintray.io/ kong-enterprise-edition`. |
+    |`image.tag` | The Docker image tag you want to pull down, e.g. `"1.5.0.2-alpine"`. |
+    |`ingressController.enabled` | Set to `true` if you want to use the Kong Ingress Controller, or `false` if you don't want to install it. |
+
+3. (Optional) If not using the Kong Ingress Controller, remove the entire section of `Ingress Controller parameters`.
+
+4. In the `Kong Enterprise` section, enable Kong Manager (`manager`) and Kong Dev Portal (`portal`).
+
+    For example:
+    ```
+    manager:
+      enabled: true
+      type: LoadBalancer
+      http:
+        enabled: true
+        servicePort: 8002
+      tls:
+        enabled: true
+        servicePort: 8445
+
+    portal:
+      enabled: true
+      type: LoadBalancer
+      http:
+        enabled: true
+        servicePort: 8003
+      tls:
+        enabled: true
+        servicePort: 8446
+    ```
+
+5. Fill in the rest of the parameters as they fit your implementation. Use the comments in the sample file to guide you, and see the documentation on [Kong Enterprise parameters](https://github.com/Kong/charts/blob/master/charts/kong/README.md#kong-enterprise-parameters) for more details.
+
+## Step 8. Deploy Kong Enterprise on Kubernetes
+The steps in this section show you how to install Kong Enterprise on Kubernetes using Helm.
+>Note: the following instructions assume that you're running Helm 3.
+
+{% navtabs %}
+{% navtab kubectl %}
+1. Run:
+    ```
+    $ helm install  my-kong kong/kong --namespace kong --values ~/values.yaml
+    ```
+    This may take some time.
+
+2. Check pod status:
+    ```
+    $ kubectl get pods -n kong
+    ```
+    Once migrations are complete and the `my-kong-kong-<ID>` pod is running, move on to the next section.
+{% endnavtab %}
+{% navtab OpenShift oc %}
+1. Run:
+    ```
+    $ helm install  my-kong kong/kong --namespace kong --values ~/values.yaml
+    ```
+    This may take some time.
+
+2. Check pod status:
+    ```
+    $ oc get pods -n kong
+    ```
+    Once migrations are complete and the `my-kong-kong-<ID>` pod is running, move on to the next section.
+{% endnavtab %}
+{% endnavtabs %}
+
+## Step 9. Finalize Configuration and Verify Installation
+{% navtabs %}
+{% navtab kubectl %}
+1. Run:
+    ```
+    $ kubectl get svc -n kong my-kong-kong-admin --output=jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    ```
+
+2. Copy the IP address from the output, then edit the `values.yaml` file to add the following line under `env` section:
+
+    ```
+    admin_api_uri: <your-ip>
+    ```
+
```

Comment:
This will likely cause issues with RBAC: there's no way to configure the admin session configuration such that it can set a cookie on one IP and use it on another. We should add a note here that users should set up DNS and update these settings to use hostnames at their earliest convenience, and that they must do so before enabling RBAC.

We should provide documentation to both encourage users to use hostnames and explain how they relate to one another in the context of Kong, but that's fairly involved and I think we're best off with a brief note here for now to get the doc released quickly. This cuts across both Kubernetes and non-Kubernetes installations, though on Kubernetes the Admin and Manager IPs will differ since they're separate Services (this isn't the case if you're accessing them through an Ingress, but in that case you'd be using hostnames+paths rather than IPs). Jira ticket DOCS-1054 describes that proposed document in more detail.

Reply:
> We should add a note here that users should set up DNS and update these settings to use hostnames at their earliest convenience, and that they must do so before enabling RBAC.

So we could go one of two ways here: 
- Keep the instructions as they are and add a note as you suggest, saying that this won't work in production and they need to set up a DNS, update this setting again, then optionally enable RBAC, or;
- Change the steps to tell users to set up a DNS before doing this, which becomes something like:

2. Copy the IP address from the output and set up a DNS pointing to that IP. 
- This is something you do outside of Kong, right? So we won't be telling them how to do it.
3. Set `admin_api_uri` to point to <DNS(:port?)>

Reply:
Correct, though for the second option there are a several settings that interact with one another and have to be chosen to meet CORS and cookie-sending requirements. That won't easily fit into a brief paragraph here, so I think the best option is to briefly mention that you definitely need DNS hostnames for RBAC now and write something more detailed later. It's not perfect, but it's not worse than the status quo.